### PR TITLE
supply usdc cap

### PR DESCRIPTION
**Pull request by shan-57blocks**
image

### CODE REVIEW

**Comment by PlayJok3r** [comment]
The information is not clear enough. How about "Available liquidity capacity is xxxx, your balance is xxx.
"

File: packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx [hunk 1]
```diff
@@ -113,12 +129,32 @@ export function ChooseAmount({
...
+      selectedTranche === 'junior' ? juniorAvailableCapBN : seniorAvailableCapBN
+    const available = balance.gt(trancheCap) ? trancheCap : balance
+    const infos: string[] = []
+    infos.push(
+      `${formatNumber(
+        ethers.utils.formatUnits(balance, decimals),
+      )} balance, ${formatNumber(
+        ethers.utils.formatUnits(trancheCap, decimals),
+      )} liquidity available`,
+    )
```
Comment (mliu) [inline packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx:143]: Design of this looks a little weird. Maybe we render is as a grid like we used to have on this screen?

Your Balance                                               5,000 USDC
Tranche Space Remaining                         12,000 USDC

**Review by mliu — APPROVED** [review]

File: packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx [hunk 2]
```diff
@@ -113,12 +129,32 @@ export function ChooseAmount({
...
+    if (campaign && campaignPoints > 0) {
+      infos.push(
+        `You will earn ${formatNumber(
+          campaignPoints,
+        )} Huma points for the entire ${lockupPeriodMonths} months.`,
+      )
```
Comment (mliu) [inline packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx:149]: Looks a little too squeezed to me. Can we align this to the bottom of the modal, right above the button?

File: packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx]: Updated

**Comment by shan-57blocks** [comment]
> The information is not clear enough. How about "Available liquidity capacity is xxxx, your balance is xxx. "

Updated as Michael's comment

File: packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx (hunk 2 above)
Comment (shan-57blocks) [reply packages/huma-widget/src/components/Lend/supplyV2/3-ChooseAmount.tsx]: Please check the updated image.

**Pull request merged**